sdk/provisioning/Azure.Provisioning/api/Azure.Provisioning.net461.cs

```diff
@@ -0,0 +1,466 @@
+namespace Azure.Provisioning
```

Comment:
How is this one property making things suddenly multitarget?

Reply:
Yeah I'm not sure what happened here. We were already multitargeting, but somehow the api.cs files were not created for the other targets.